src/seedu/addressbook/AddressBook.java

```diff
@@ -118,15 +118,15 @@
 
     private static final String COMMAND_CLEAR_WORD = "clear";
     private static final String COMMAND_CLEAR_DESC = "Clears address book permanently.";
-    private static final String COMMAND_CLEAR_EXAMPLE = COMMAND_CLEAR_WORD;
+//    private static final String COMMAND_CLEAR_EXAMPLE = COMMAND_CLEAR_WORD;
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.